### PR TITLE
Ticket659: Replaced PPH-specific assets

### PR DESCRIPTION
**Pull request by baylie-r-g**
### Description
Created a new WTix logo to replace PPH logos. Link in admin navbar leads back to home
I found a more general Creative Commons photo to place on the banner, no licensing or credit is required for commercial or personal use. (no copyright issues)

Replaced as many PPH logos as I could in the ticketing manager. Unable to find line holding the image in the tab.

### Risks
Unable to find the line for the final logo in the Ticket Management tree

### Validation
Ran all main pages and admin pages of the WTix site. I was able to check the loading page as well.

### Issue
(https://github.com/WonderTix/WonderTix/issues/659)

### Operating System
Windows 11

### CODE REVIEW

File: client/tailwind.config.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ module.exports = {
 
     extend: {
       backgroundImage: {
-        'hero': 'url(../src/assets/portland_playhouse_cover.jpg)',
+        'hero': 'url(../src/assets/pexels-cottonbro-6896179.jpg)',
```
Comment (ben16williams) [inline client/tailwind.config.js:21]: I was thinking that we could make this even more generic. So something like a solid background color or a simple pattern that's on theme. Ideally Wondertix will be general enough that it wont be restricted to servicing playhouses. I also don't know there are copy right issues with this image?

File: client/public/index.html [hunk 2]
```diff
@@ -8,8 +8,8 @@
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="icon" href="2011_Logo_white.png" />
-    <link rel="apple-touch-icon" href="2011_Logo_white.png" />
+    <link rel="icon" href="WTix logo light.png" />
+    <link rel="apple-touch-icon" href="WTix logo light.png" />
```
Comment (ben16williams) [inline client/public/index.html:12]: Probably a non-issue but not having any spaces in the image file name would be on the safer side

File: package.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "react-swipeable-views": "^0.14.0"
+  }
+}
```
Comment (ben16williams) [inline package.json:5]: Is this really needed at the root level?

**Comment by ben16williams** [comment]
Would you mind adding a little more descriptive title? It helps when looking back through old PRs

**Comment by ben16williams** [comment]
I see that there are still a few reference to Portland Playhouse in the code-base that we'll want to remove. You should be able to do a search and find with most text editors (including vscode) to find the rest

File: client/tailwind.config.js (hunk 1 above)
Comment (baylie-r-g) [reply client/tailwind.config.js:21]: The image I placed there is Creative Commons so it is free to use in any way. I can switch to something more generic though

File: package.json (hunk 3 above)
Comment (baylie-r-g) [reply package.json:5]: I didn't add this myself so I do not know if it is needed at root level or not

**Comment by baylie-r-g** [comment]
New commits for this PR

File: client/public/index.html
Comment (baylie-r-g) [inline client/public/index.html:1]: Should I add %PUBLIC_URL% back into this file?

File: package.json (hunk 3 above)
Comment (henrykaus) [reply package.json:5]: @benfaustin95 Do you know why this is showing up now? I believe you were using the swipeable-views at one point

Comment (benfaustin95) [reply package.json:5]: The package is still being used at the client level, but shouldn't be at the root level. It looks like the package.json is being added with this PR is it possible the package was installed at the wrong level which generated the the file?

Comment (henrykaus) [reply package.json:5]: Ok, in that case, the file should probably just be deleted from the root directory.

File: client/public/index.html
Comment (henrykaus) [reply client/public/index.html:1]: Yeah, I think that would be best

File: package-lock.json
Comment (henrykaus) [inline package-lock.json:1]: This file should also be deleted since it shouldn't be around at the root level

**Comment by baylie-r-g** [comment]
New push created. Need to know if there is a server to upload a new logo so that it can be changed on the auth0 login

File: client/src/components/Ticketing/ticketingmanager/udash_navbar.tsx
Comment (henrykaus) [inline client/src/components/Ticketing/ticketingmanager/udash_navbar.tsx:1]: This file was removed in a recent PR, so you can delete this file.

File: client/src/components/CRM/Navigation.tsx
Comment (henrykaus) [inline client/src/components/CRM/Navigation.tsx:1]: This file can also be removed

**Review by henrykaus — COMMENTED** [review]
Two minor changes and then this looks good to go. I really like the new WonderTix logo!!

**Review by henrykaus — APPROVED** [review]
Thanks for all the changes! Feel free to merge once tests pass.

**Pull request merged**